elftools/common/construct_utils.py

```diff
@@ -110,3 +111,34 @@ def _build(self, obj, stream, context):
         context[self.name] = stream.tell()
     def _sizeof(self, context):
         return 0
+
+_UBInt24_packer = Struct(">BH")
+_ULInt24_packer = Struct("<HB")
+
+class UBInt24(StaticField):
+    """unsigned, big endian 24-bit integer"""
+    def __init__(self, name):
+        StaticField.__init__(self, name, 3)
+
+    def _parse(self, stream, context):
+        global _UBInt24_packer
```

Comment:
I'm surprised you need this `global` here and elsewhere

Reply:
Technically not needed, more of a hint to self and other maintainers. Might work as a microoptimization, too. Should I remove them?